### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Matt1994**
Iteration 1

### CODE REVIEW

**Review by mavagio — COMMENTED** [review]
Overall the code looks good, however the documentation is very minimal and the pdf creation is entangled with many hard coded numbers and strings. Without explanation of function parameters it will become very hard in the future to change the design of the pdf.

Dom element lookups are expensive (e.g $(document) or $('#fields') ). Now you have a few of them but in the future they will increase, especially when you try to check user input. Cache the dom instead and have a method for binding events and unbinding them. This can be a good step towards modular js. There are many tutorials that show how to make js modular, I will link one below, it tackles this problem nicely with example:
https://youtu.be/HkFlM73G-hk?list=PLoYCgNOIyGABs-wDaaxChu82q_xQgUb4f

Alternatively you can use frameworks for that. You can consult with your customer to check on his preference. Consider that this project can be a single page application. 

There are many magical strings and hard coded numbers, as mentioned before, in the card.js and document.js. I see that they are used for styling the pdf but it makes very hard for the future to change the style this way. Separation would make things better. Perhaps you can have json config file for it. Separating strings will also help to translate them easily if needed. 

Small remarks:
Pay attention on formatting:
-I see some inconsistent tabs and indentations. (e.g. card.js lines 4,16,37... )
-A lot of white spaces in html and js files. It feels clumsy and hurts readability.
-When commenting methods, use JavaDoc equivalent comments for JS (JSDoc e.g. /** */).
-Keep the file structure consistent: Don’t have an image in the styles folder.

File: create_card.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+	
+<head>
+		<meta charset="utf-8">
```
Comment (mavagio) [inline create_card.html:5]: inconsistent tabs.

File: js/card.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+//Adds a field HTML row in the table
+function addField() {
+	$('#fields').append("<tr><td><input class='label' maxlength='15' type='text' /></td><td><input class='field' maxlength='100' type='text' /></td><td><input class='removeField' type='button' value='Remove' /></td></tr>");
+	}
```
Comment (mavagio) [inline js/card.js:4]: keep the indentation consistent.

**Review by mavagio — APPROVED** [review]

**Comment by Theys96** [comment]
I agree the code quality can really use quite some improvement. This can be my job for the next iteration, since it's my original work anyway.

**Pull request merged**